### DISCUSSION TITLE
APLR

### DISCUSSION THREAD

**Discussion by DeFlanko** [xnorpx/blue-onyx#7] [Q&A]
Does this support APLR capabilities? Will it be a feature in the future if it doesn't already?

**Comment by xnorpx** [comment]
I won't spend time on it, as I don't need it. If someone want to provide a clean pr for it I would be open to merge it.

**Comment by DeFlanko** [comment]
I only ask because i was asked to UAT Blue Onyx and that's one of the features that is an option in the AI window for Blue Iris.  Trying to find an all in one solution from CPAI.

**Reply by xnorpx** [reply]
Sure that makes sense, one of my main goals with Blue Onyx is to keep it simple and avoid feature creep.

But as stated above APLR is border line useful but I personally don't have a need or hardware for it.
